style.css

```diff
@@ -0,0 +1,75 @@
+#nav-bar ul {
+    list-style-type: none ;
+    float: right;
+}
+
+#nav-bar ul li {
+    display: inline;
+    margin-right: 25px;
+    
+}
+
+#nav-bar ul li a{
+    text-decoration: none;
+    color: black;
+}
+
+#header-img{
+    float: left;
+    width: 100%;
+    height: 100%;
+    max-width: 300px;
+}
+
+#header{
+    margin-bottom: 30px;
+    height: 70px;
+    position: fixed;
+    width: 100%;
+    top: 0;
+    left: 0;
+    background-color: #fff;
+}
+
+
+/* @media only screen and (orientation: landscape) {
+    .flex-container{
+        display: flex;
+    }
+  } */
```

Comment:
I think you should try to avoid block comments in your CSS code to keep your code clean. Read more about HTML and CSS best practices [here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)